apis/v1alpha1/gatewayclass_types.go

```diff
@@ -103,11 +103,10 @@ type GatewayClassSpec struct {
 	// any additional configuration.
 	//
 	// Valid resources for reference are up to the Controller. Examples
-	// include "configmaps" (omit or specify the empty string for the group
-	// to indicate the core API group) or a custom resource (CRD).  Omitting
-	// or specifying the empty string for both the resource and group
-	// indicates that the resource is "configmaps".  If the referent cannot
-	// be found, the GatewayClass's "InvalidParameters" status condition
+	// include "configmaps"  or a custom resource (CRD).
+	//
+	// If the referent cannot be found,
+	// the GatewayClass's "InvalidParameters" status condition
```

Comment:
Nit: the line length here feels a bit off. I generally try to wrap lines at 80 chars.